store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9526,5 +9526,10 @@ TODO: delete them permanently from here
   <desc>port number on which the remote IMAP SSL server should listen</desc>
 </attr>
 
+<attr id="3017" name="zimbraReverseProxyUpstreamFairShmSize" type="integer" min="32" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="8.8.1" requiresRestart="nginxproxy">
```

Comment:
I chose integer on purpose since I am unsure if anything other than 'k' is allowed.
This removes all ambiguity and prevents me from having to compute a value for 'k' from whatever unit the user specified.